spec/models/feedback_reporters/support_reporter_spec.rb

```diff
@@ -97,5 +103,45 @@
         expect(subject.report_attributes.fetch("description")).to eq("Hi!http://example.com/Camera-icon.svgBye!")
       end
     end
+
+    context "if the report has an empty ip address" do
```

Comment:
"IP" for consistent capitalization? Or "ip_address" if you'd rather the field name.